webapp/channels/src/components/file_preview/file_preview.tsx

```diff
@@ -66,19 +66,15 @@ export default class FilePreview extends React.PureComponent<Props> {
                     imageClassName += ' normal';
                 }
 
-                let thumbnailUrl = getFileThumbnailUrl(info.id);
-                if (Utils.isGIFImage(info.extension) && !info.has_preview_image) {
-                    thumbnailUrl = getFileUrl(info.id);
-                }
-
                 previewImage = (
                     <div
                         className={imageClassName}
                         style={{
-                            backgroundImage: `url(${thumbnailUrl})`,
                             backgroundSize: 'cover',
                         }}
-                    />
+                    >
+                        <img src={getFileUrl(info.id)}/>
```

Comment:
Why change this to use an img tag instead of a background image?

If we change this, we need to make sure that images with weird dimensions are still cropped and scaled properly to fill this space. I think that was the reason that this used a background image originally, but that code was so old that we might have better ways to do that now.

Reply:
@hmhealey, I used img instead of url thinking that url() cannot execute the getFileUrl function that returns the string which is used to render the animation. However, I can use the url property by storing the value of the getFileUrl method in a variable and passing it inside url(). Also, since the file object contains the height and width information, I can use these inside img tag as well. So in that case, I can continue using img and use the height and width information from the file object. Are there any problems in this approach? Please let me know